src/file-upload/file-upload.tsx

```diff
@@ -93,6 +93,27 @@ export const FileUpload = ({
     // =========================================================================
     // RENDER FUNCTIONS
     // =========================================================================
+    const renderInfo = () => {
+        if (!title && !description) {
+            return null;
+        }
+
+        return (
+            <TitleContainer>
+                {typeof title === "string" ? (
+                    <Title weight="regular">{title}</Title>
+                ) : (
+                    <div>{title}</div>
+                )}
+                {typeof description === "string" ? (
+                    <Description weight="semibold">{description}</Description>
+                ) : (
+                    <div>{description}</div>
+                )}
```

Comment:
if either `title` of `description` is undefined, empty divs will still be rendered

Reply:
whoops thanks for spotting, restored the original falsy check